<launch>
    <arg name="namespace" default="yk_creator"/>
    <group ns="$(arg namespace)">
        <node pkg="pallet_pick_and_place" name="save_lego_pose" type="save_lego_pose.py" output="screen">
            <param name="namespace" value="$(arg namespace)"/>
            <param name="root_pwd" value="$(find pallet_pick_and_place)"/>
        </node>
    </group>
</launch>
